### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ztelur**
**What this PR does**:

update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,23 @@ English | [中文](./README_CN.md)
 
 # Introduction
 
-**Dubbo-Go-Pixiu**(official site: https://dubbo-go-pixiu.github.io/) is a gateway that mainly focuses on providing gateway solution to your Dubbo and RESTful services.
+**Dubbo-Go-Pixiu**(official site: https://dubbo-go-pixiu.github.io/) is a high-performance API gateway and multi-language solution Sidecar in the Dubbo ecosystem
+
+
+![](https://dubbo-go-pixiu.github.io/img/pixiu-dubbo-ecosystem.png)
+
+It is an open source Dubbo ecosystem API gateway and language solution for accessing dubbo clusters. As an API gateway, Pixiu can receive external network requests, convert them into dubbo and other protocol requests, and forward them to the back cluster; as a sidecar, Pixiu expects to register to the Dubbo cluster instead of the proxy service, allowing multilingual services to access the Dubbo cluster to provide faster solution.
```
Comment (AlexStocks) [inline README.md]: It is an open source Dubbo ecosystem API gateway, and also a sidecar to let other compute language program access the dubbo clusters by HTTP/gRPC protocol.

Comment (ztelur) [reply README.md]: replaced!

**Review by PhilYue — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -78,78 +85,24 @@ curl -X POST 'localhost:8881/api/v1/test-dubbo/user' -d '{"id":"0003","code":3,"
...
 #### 
 ```shell
-docker pull phial3/dubbo-go-pixiu:latest
+docker run --name pixiug-gateway -p 8888:8888 dubbogopixiu/dubbo-go-pixiu:latest
+
 ```
 ```
-docker run --name pixiuname -p 8883:8883 \
+docker run --name pixiug-gateway -p 8888:8888 \
     -v /yourpath/conf.yaml:/etc/pixiu/conf.yaml \
```
Comment (MasterKenway) [inline README.md]: wrong spell ?

**Review by MasterKenway — APPROVED** [review]

**Review by mark4z — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ztelur) [reply README.md]: fix

**Pull request merged**